.github/workflows/preview-release.yml

```
@@ -133,12 +133,14 @@ jobs:
       run: dotnet build "${{ github.workspace }}/${{ env.PROJECT_NAME }}/${{ env.PROJECT_NAME }}.csproj" `
         -c ${{ env.BUILD_CONFIG }} `
         --no-restore
```

Add a back tick to the end of this line for multiline PowerShell

```suggestion
        --no-restore `
```